scripts/config/attr_gen_camels.R

```diff
@@ -28,32 +40,40 @@ main <- function(){
     lapply( function(x) gsub(pattern = "Gage_", replacement = "",x=x)) |>
     unlist()
 
-  utils::write.table(gage_ids,glue::glue('{home_dir}/noaa/camels/gagesII_wood/camels_ii_gage_ids.txt'),row.names = FALSE,col.names = FALSE)
```

Comment:
in general, arguments to a function should be separated by a space after each ','... e.g., write.table(xxxx, row...., col...)